src/urls/WebsiteURIs.js

```diff
@@ -4,7 +4,7 @@ import { Link, Redirect } from "react-router-dom";
 import { AVAILABLE_LANGUAGES } from "../i18n";
 import { generatePath } from "react-router-dom";
 
-const ORCID_OAUTH_CLIENT_ID = "APP-I3S9GLKWPY01BYEA";
+const ORCID_OAUTH_CLIENT_ID = "APP-ZOEBA1AI4XE8F82W";
```

Comment:
Is this temporary for testing purposes? I think we should leave the original one in

Reply:
Agreed - this key is actually used in the main deployment. It'll be rotated once we go to prod but it's good to keep the build artifacts deployable as-is.

BTW any ideas on how to pass that value into react from env vars maybe?
If you know of any sensible patterns that could be used here I'm open to suggestions :)